app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
@jcoyne ok. I see where you're going, but I thinking we're creeping out of scope. I agree that we will need a finer-grained idea of "administrator" but I'm interpreting the present scenario as a local root user. In other words, this is a user specific to the application, not to Fedora, that just needs to do anything. Even with multiple levels of administrators, aren't you still going to need a super user whose rights trump all?